pulsar-client-1x-base/pulsar-client-1x/src/main/java/org/apache/pulsar/client/api/ClientConfiguration.java

```diff
@@ -158,7 +158,7 @@ public int getIoThreads() {
     }
 
     /**
-     * Set the number of threads to be used for handling connections to brokers <i>(default: 1 thread)</i>.
+     * Set the number of threads to be used for handling connections to brokers <i>(default is Runtime.getRuntime().availableProcessors())</i>.
```

Comment:
The module `pulsar-client-1x` is deprecated. You can make your changes here: https://github.com/apache/pulsar/blob/c172a775fa0fc5762a9703669ed8ebcd2efbe042/pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java#L245-L263

Reply:
I was careless, it has been corrected.